src/ui.tsx

```
@@ -504,20 +511,77 @@ function VisualizationOptions({ state }: { state: State }) {
+        method: 'post',
+        headers: {
+          Authorization: `Bearer ${state.svgIoOptions.apiKey}`,
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ prompt: state.svgIoOptions.prompt, style: state.svgIoOptions.vecType, negativePrompt: '' })
+      });
+      dispatch({ type: "SET_SVGIO_OPTION", value: { status: 'Loading ...' } })
+      const imageUrl = (await apiResp.json()).data[0].svgUrl;
+      const imgResp = await fetch(imageUrl);
+      const imgData = await imgResp.text();
+      dispatch(setPaths(readSvg(imgData)));
+    } catch (error) {
+      console.error(error);
+    } finally {
+      dispatch({ type: "SET_SVGIO_OPTION", value: { status: '' } });
+    }
+  }
+  return <>
+    <div>
+      <label title="Need an API Key">svg.io API Key
+        <input type="text" value={state.svgIoOptions.apiKey}
+          onChange={(e) => dispatch({ type: "SET_SVGIO_OPTION", value: { apiKey: e.target.value } })} />
+      </label>
+      <label>Type
+        <select value={state.svgIoOptions.vecType} 
+          onChange={(e) => dispatch({ type: "SET_SVGIO_OPTION", value: { vecType: e.target.value } })}>
+          <option value={"FLAT_VECTOR"}>Flat</option>
+          <option value={"FLAT_VECTOR_OUTLINE"}>Outline</option>
+          <option value={"FLAT_VECTOR_SILHOUETTE"}>Silhouette</option>
+          <option value={"FLAT_VECTOR_ONE_LINE_ART"}>One Line Art</option>
+          <option value={"FLAT_VECTOR_LINE_ART"}>Line Art</option>
+        </select>
+      </label>
+      <label title="propmpt">Prompt
+        <textarea value={state.svgIoOptions.prompt} size={"3"}
+          onChange={(e) => dispatch({ type: "SET_SVGIO_OPTION", value: { prompt: e.target.value } })}>
+        </textarea>
+      </label>
+    </div>
+    <div>
+      {state.svgIoOptions.status ? <span>{state.svgIoOptions.status}</span> : <button onClick={generateImage}>Generate!</button>}
```

This button should be disabled with no API key, otherwise it appears to do something when clicked and fails silently.